linkerd/docs/telemetry.md

```diff
@@ -103,3 +103,30 @@ Key        | Default Value | Description
 ---------- | ------------- | -----------
 sampleRate | _required_    | What percentage of traces to record.
 capacity   | 10            | The maximum number of recent traces to store
+
+## Usage
+
+> Example usage config
+
+```yaml
+telemetry:
+ - kind: io.l5d.commonMetrics
+ - kind: io.l5d.usage
+```
+
+kind `io.l5d.usage`
+
+In order to make improvements and prioritize features, we'd like to gain a
+broader picture of how users run linkerd. This telemeter helps us by gathering anonymized usage data,
```

Comment:
It might be nice to explicitly call out certain data that will NOT be collected (labels, dtabs, service addresses, request data of any kind, etc.)